helm/cloud-provider-cloud-director/charts/cloud-director-named-disk-csi-driver/templates/controllerplugin-statefulset.yaml

```diff
@@ -39,6 +39,7 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --timeout=300s
+            - --default-fstype=ext4

```

Comment:
out of curriosity - what's the fstype if this flag isn't set?

Reply:
And a follow-up: doesn't that fstype respect fs groups?

Reply:
I'm not entirely sure to be honest I need to do more reading but this comes from here.
Also this flag is set in CAPO.

https://github.com/vmware/cloud-director-named-disk-csi-driver/issues/64
https://github.com/vmware/cloud-director-named-disk-csi-driver/pull/74